src/main/java/com/conveyal/datatools/manager/controllers/api/NoteController.java

```diff
@@ -151,11 +151,9 @@ public static Note createNote (Request req, Response res) throws IOException {
             // Send notifications to comment subscribers.
             // TODO: feed-commented-on has been merged into feed-updated subscription type. This should be clarified
             // in the subject line/URL of the notification email.
-            NotifyUsersForSubscriptionJob.createNotification(
-                    "feed-updated",
-                    feedSource.id,
-                    message
-            );
+            NotifyUsersForSubscriptionJob notifyUsersForSubscriptionJob
+                = new NotifyUsersForSubscriptionJob("feed-updated",feedSource.id,message);
+            notifyUsersForSubscriptionJob.run();
```

Comment:
Use the light executor for this. See https://github.com/ibi-group/datatools-server/pull/348#discussion_r512337741 for reasoning.

Reply:
Scratch that, I think the previous way should be used. Please revert due to reasons list in https://github.com/ibi-group/datatools-server/pull/348#discussion_r512341045.

Reply:
Updated.